src/gasman.c

```diff
@@ -1805,82 +1799,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
I have no memory of why, but looking at the code it seems there was a version which called the C level finalisers  in a separate pass before the sweep pass, and that this was later integrated with the sweep pass. At a quick glancce, no C level finalisers are ever set anyway, so, unless it's used in libsing or something, that whole feature could be dropped.